### ISSUE TITLE
Cannot login

### ISSUE THREAD

**Issue by garyhazzard** [desktop/desktop#9287]
Hi

can you help please?

I cannot login. when trying i get and authentication error.

I have updated the password in gitlab.

any help would be appreciated

login: gary.hazzard@lb-solutions.com

### Version & OS

Open 'About GitHub Desktop' menu to see the Desktop version. Also include what operating system you are using.

### Actual behavior

A clear and concise description of what actually happened.

### Screenshots

Add screenshots to help explain your problem, if applicable.

### Logs

Attach your logs by opening the `Help` menu and selecting `Show Logs...`, if applicable.

**Comment by steveward** [comment]
Thanks for the report @garyhazzard. Could you upload the log file from GitHub Desktop so that I could get some more information about this error? To access the log files go to the file menu in GitHub Desktop and select `Help` > `Show Logs in Finder (macOS) or Explorer (Windows)`.

The log files are created daily -- please upload a log file as an attachment from a day where you experienced the issue. Thanks!

**Comment by garyhazzard** [comment]
Hi Steve
 
These are the logs.
 
Kindest Regards
 
Gary
 
Athena
Support: +44 1342 833 005
Mobile: 07753 744 199
Web:  <http://www.lb-solutions.com/> www.lb-solutions.com
 
 

 
 
 
The information in this email and any attachments may be confidential and intended for the sole use of the addressee(s). Access, copying, disclosure or re-use, in any way, of the information contained in this email and attachments by anyone other than the addressee(s) is unauthorised. 

We accept no liability for the email (or for any attachments). If you have received this message in error, please advise the sender immediately by reply e-mail, immediately and permanently delete it and do not use, copy or disclose it.

**Comment by steveward** [comment]
@garyhazzard it looks like your log file didn't make it through as an attachment -- could you try uploading it directly in the issue thread in your browser by dragging-and-dropping it into a comment?

**Comment by garyhazzard** [comment]
[2020-03-05.desktop.production.log](https://github.com/desktop/desktop/files/4339849/2020-03-05.desktop.production.log)
[2020-03-06.desktop.production.log](https://github.com/desktop/desktop/files/4339850/2020-03-06.desktop.production.log)
[2020-03-12.desktop.production.log](https://github.com/desktop/desktop/files/4339851/2020-03-12.desktop.production.log)
[2020-03-13.desktop.production.log](https://github.com/desktop/desktop/files/4339852/2020-03-13.desktop.production.log)
[2020-03-15.desktop.production.log](https://github.com/desktop/desktop/files/4339853/2020-03-15.desktop.production.log)
[2020-03-16.desktop.production.log](https://github.com/desktop/desktop/files/4339854/2020-03-16.desktop.production.log)

**Comment by garyhazzard** [comment]
Hi Steve
 
I have loaded them in the comment area.
 
Did you get them
 
 
 
 
 
Kindest Regards
 
Gary
 
Athena
Support: +44 1342 833 005
Mobile: 07753 744 199
Web:  <http://www.lb-solutions.com/> www.lb-solutions.com
 
 

 
 
 
The information in this email and any attachments may be confidential and intended for the sole use of the addressee(s). Access, copying, disclosure or re-use, in any way, of the information contained in this email and attachments by anyone other than the addressee(s) is unauthorised. 

We accept no liability for the email (or for any attachments). If you have received this message in error, please advise the sender immediately by reply e-mail, immediately and permanently delete it and do not use, copy or disclose it.

**Comment by steveward** [comment]
@garyhazzard thanks for uploading those. We have a specific guide that walks through how to authenticate using GitHub Desktop:

https://github.com/desktop/desktop/blob/development/docs/integrations/gitlab.md

If you run into any issues with that please let me know!

**Comment by garyhazzard** [comment]
Hi Steve
 
Thanks, it is helpful.
 
The issue I have is that when trying to clone the object in Github it will not me login. It give the error that my password is incorrect even though I changed it in GitLab immediately before attempting the clone in Github.
 
Any suggestion would be helpful.
 
Kindest Regards
 
Gary
 
Athena
Support: +44 1342 833 005
Mobile: 07753 744 199
Web:  <http://www.lb-solutions.com/> www.lb-solutions.com
 
 

 
 
 
The information in this email and any attachments may be confidential and intended for the sole use of the addressee(s). Access, copying, disclosure or re-use, in any way, of the information contained in this email and attachments by anyone other than the addressee(s) is unauthorised. 

We accept no liability for the email (or for any attachments). If you have received this message in error, please advise the sender immediately by reply e-mail, immediately and permanently delete it and do not use, copy or disclose it.

**Comment by steveward** [comment]
>The issue I have is that when trying to clone the object in Github it will not me login. It give the error that my password is incorrect even though I changed it in GitLab immediately before attempting the clone in Github.

You cannot directly log in to your GitLab account in GitHub Desktop -- you will need to generate a token by following the steps in the article I linked. You can then enter your GitLab username and token, which will be cached and allow you to clone repositories from GitLab. Let me know if you run into any issues with that.

**Comment by dtistaley** [comment]
Hello @steveward. I know this issue is closed, but I was actually having the seemingly same issue as Gary was above. I'll provide more information on the issue specifically below:

**Summary:**
User is unable to clone an external project from GitLab to Github Desktop. Additionally, when receiving the `Authentication Failed` window, there isn't a place to enter a username or password for the external source, GitLab.

**Detailed Information**
A coworker of mine was having this issue and I was able to replicate it by creating a new user account on Github, downloading and installing GitHub Desktop, and then attempting to clone a project from GitLab using the same directions provided in the guide below:

  https://github.com/desktop/desktop/blob/development/docs/integrations/gitlab.md

I have also created an Access Token from GitLab in preparation for the external login mentioned in your guide. In execution, I found the authentication error no longer includes a place to enter username or password login information as the guide mentions. Instead, this is the window that popped up when the authentication failed:

![image](https://user-images.githubusercontent.com/65295240/81844985-e8c47200-951d-11ea-8561-f50108024ec8.png)

I wonder if the original feature of being able to connect to an external source, such as GitLab, has been removed/disabled? Or maybe there is something that changed in how to go about this?
